packages/blade/src/components/Chip/_decisions/decisions.md

```diff
@@ -0,0 +1,144 @@
+# Chip Decisions <!-- omit in toc -->
+
+Chips embody a collection of selectable entities that empower users to engage in choices, sift through content, and invoke pertinent actions.
+
+This document outlines the API details of the `Chip` and `ChipGroup` components, encompassing their structural composition, functional attributes, and visual representation.
+
+<img width="100%" src="./chip-variants.png" alt="Different variants of the Chip component">
+
+- [Design](#design)
+- [`Chip` API](#chip-api)
+- [`ChipGroup` API](#chipgroup-api)
+- [Examples](#examples)
+  - [Basic/Uncontrolled Usage](#basicuncontrolled-usage)
+  - [Controlled Usage](#controlled-usage)
+- [Accessibility Considerations](#accessibility-considerations)
+- [Open Questions](#open-questions)
+
+## Design
+
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?type=design&node-id=52377%3A27686&mode=design&t=pFzCay0x4a6sTcoQ-1) to all variants of the `Chip` component.
+
+## `Chip` API
+
+> **Note**
+>
+> The `Chip` component is inherently tied to the `ChipGroup` and cannot be utilized outside its context.
+
+| Prop                    | Type                                   | Default     | Description                                                                                                                                        | Required |
+| ----------------------- | -------------------------------------- | ----------- | -------------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| **children**            | `React.ReactText \| React.ReactText[]` | `undefined` | Renders the label text for the Chip                                                                                                                | ✅       |
+| **icon**                | `Icon`                                 | `undefined` | Displays the Blade Icon component within the Chip                                                                                                  | ❌       |
```

Comment:
Trailing icon not supported?

<img width="683" alt="Screenshot 2023-08-22 at 3 51 08 PM" src="https://github.com/razorpay/blade/assets/29686866/8aab6839-9786-44cf-bb0c-177f87abf515">

[Design][1]

[1]: https://www.figma.com/file/xZP4W0zCEVyjpxn12XtQ96/GPT-for-lead-generation?type=design&node-id=561-171566&mode=design&t=18xlkeF6bSuCXqX6-4

Reply:
@cseas For consistency we will support leading icons only, the designs will need an update.